### ISSUE TITLE
A11y: missing alts attributes

### ISSUE THREAD

**Issue by felixzapata** [PolymerElements/app-layout#356]
### Description

There are no `alt` attributes in the images.

### Expected outcome

Every image used in the example should have an `alt` attribute.
